docs/docs/integrate/embedding.md

```diff
@@ -266,12 +271,46 @@ The API accepts the following parameters:
 
 | Parameter | Description                                                                                                                                                                                     | Required                         |
 | --- |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------|
-| resource | The name of the dashboard to embed                                                                                                                                                              | Yes                              |
-| user_email | The email of the user to embed the dashboard for                                                                                                                                                | No (either this or `attributes`) |
-| attributes | Json payload to be put in the access token, used to pass attributes to the dashboard for enforcing policies. When using this make sure to pass all the attributes used in your security policy like `email`, `domain` and `admin` | No (either this or `user_email`) |
-| ttl_seconds | The time to live for the iframe URL                                                                                                                                                             | No (Default: 86400)              |
+| resource | The name of the dashboard to embed                                                                                                                                              | No (if not specified, `navigation` should be set to `true`)                              |
+| navigation | Boolean whether to enable navigation and allow users to navigate to other dashboards (set to `false` if you wish to hard embed and allow access to only a single dashboard)                                                                                                                           | No (either set to `true` or `false`) |
```

Comment:
The default value is effectively `false`

Reply:
I'm not exactly sure I get this. Would you mind elaborating?

Reply:
Actually I think I understand. Navigation defaults to `false` unless otherwise specified.